data/Telemetry/update.info

```diff
@@ -0,0 +1 @@
+https://gist.githubusercontent.com/FadeMind/2d58a7dc5c6ad33a4a45b7e8d2349fdd/raw/70dcb743f712c9ffe7feaa458c84360170051141/win.telem.merged.hosts
```

Comment:
A question Tomasz @FadeMind,

If you update that gist, does this url for `update.info` change?   I'm wondering if there is a static URL to the RAW version of the latest gist...